src/test/enginesynctest.cpp

```diff
@@ -2257,7 +2259,7 @@ TEST_F(EngineSyncTest, SeekStayInPhase) {
     ProcessBuffer();
 
     // We expect to be two buffers ahead in a beat near 0.2
-    EXPECT_DOUBLE_EQ(0.050309901738473183,
+    EXPECT_DOUBLE_EQ(0.050309901738473162,
```

Comment:
The comment says "near" but we are using an equality comparison with hardcoded expected values?

Reply:
Yes, this value is experimental determined. The exact recalculation is missing, but that is out of scope of this PR.